modules/ROOT/pages/async-scope-reference.adoc

```diff
@@ -47,13 +47,47 @@ Note that even though the Async scope receives a copy of the Mule message, the p
 
 In other words, if the payload of your message is a mutable object (for example, a bean with different fields in it) and a message processor in your Async scope changes the value of one of the fields, the message processors outside of the Async scope see the changed values.
 
-This example shows the XML for the Async scope.
+
+== Example Async Scope Configuration
+
+The following XML fragment shows an example configuration of an Async scope inside an application flow. The Async scope contains a `file:read` operation that executes asynchronously once triggered, while the flow continues processing the next operation in the flow which is `http:request`:
```

Comment:
Suggestion: while the flow continues processing the next operation in the flow, 'http:request`:

If you leave what you have, add a comma before "which"